src/Workspaces/CSharp/Portable/Editing/CSharpImportAdder.cs

```diff
@@ -41,11 +40,12 @@ public CSharpImportAdder()
             return null;
         }
 
-        protected override SyntaxNode MakeSafeToAddNamespaces(SyntaxNode root, IEnumerable<INamespaceOrTypeSymbol> namespaceMembers, IEnumerable<IMethodSymbol> extensionMethods, SemanticModel model, Workspace workspace, CancellationToken cancellationToken)
+        protected override void AddPotentiallyConflictingImports(
+            SemanticModel model, SyntaxNode container, ImmutableArray<INamespaceSymbol> namespaceSymbols,
```

Comment:
`container` is usually the entire file unless we're adding usings inside a namespace statement or something, right?

Reply:
correct.  container is will be determined by your option about where imports go and where the current imports are (if any).  it's normally the root of the file, or the top level namespace.

i'll doc.

Reply:
doc'ed.